tests/model/test_model.py

```diff
@@ -481,6 +481,7 @@ def test__stream_info_from_subscriptions(self, initial_data, streams,
         assert all(msg_id == msg['stream_id'] for msg_id, msg in by_id.items())
         assert muted_streams == muted
         assert pinned == []  # FIXME generalize/parametrize
+        print(unpinned)
```

Comment:
@amanagr pinging you if you missed this :)

Reply:
The 2 files here, `test_model` and `test_ui_tools` here are adjustments due to the previous commits introduced right? Maybe we can move them to a new commit (Just a suggestion).